doc/appendix-at-rksv/communication/communication.md

```diff
@@ -1,6 +1,6 @@
 ## Communication
 
-This chapter expands on the descriptions of the ways to communicate with the fiskaltrust.Middleware covered in the Chapter ["Communication"](../../general/communication/communication.md) of the General Part, with country specific information applicable to the Austrian market.
+This chapter expands on the descriptions of the ways to communicate with the fiskaltrust.Middleware covered in the Chapter ["Communication"](../../general/communication/communication.md) of the General Part, with country-specific information applicable to the Austrian market.
```

Comment:
this chapter expands more on describing the ways to communicate..

Reply:
brilliant!